src/commands/nickname.ts

```diff
@@ -0,0 +1,279 @@
+import {
+    SlashCommandBuilder,
+    SlashCommandStringOption,
+    SlashCommandSubcommandBuilder,
+    SlashCommandUserOption
+} from "@discordjs/builders";
+
+import {
+    CommandInteraction,
+    Client,
+    GuildMember,
+    User,
+    CacheType,
+    MessageActionRow, MessageButton, MessageComponentInteraction
+} from "discord.js";
+
+import {ApplicationCommand, CommandPermission, CommandResult, UserInteraction} from "./command";
+import {getConfig} from "../utils/configHandler";
+import Nicknames from "../storage/model/Nickname";
+import {isTrusted} from "../utils/userUtils";
+
+const config = getConfig();
+
+enum Vote {
+    YES,
+    NO
+}
```

Comment:
ufff enums. Nimm Union Types.. 

```suggestion
type Vote = "YES" | "NO";
```

Reply:
Oder noch besser:
```suggestion
type Vote = boolean;
```